.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### JIRA link
```

Comment:
I don't think this applies to an application such as this java client. I would use this one:

https://github.com/hmcts/idam-java-client/blob/master/.github/PULL_REQUEST_TEMPLATE.md